packages/components/src/Actions/Actions.component.js

```diff
@@ -85,9 +108,13 @@ Actions.propTypes = {
 		PropTypes.oneOfType([
 			PropTypes.shape(Action.propTypes),
 			PropTypes.shape({
-				displayMode: PropTypes.oneOf([TYPE_DROPDOWN]),
+				displayMode: TYPE_DROPDOWN,
```

Comment:
Oh, sorry, it was usefull before my last fix (https://github.com/Talend/ui/pull/343#pullrequestreview-30278497). I'll delete it.